server/channels/api4/license_local.go

```diff
@@ -54,7 +54,9 @@ func localAddLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 	defer file.Close()
 
 	buf := bytes.NewBuffer(nil)
-	io.Copy(buf, file)
+	if _, err := io.Copy(buf, file); err != nil {
+		c.Logger.Warn("Failed to copy file content to buffer", mlog.Err(err))
```

Comment:
Failing to copy the input puffer seems like a blocking error to me. I think we should return an error to the client. Is there an existing error ID that we can reuse?